### PR TITLE
Switching jewish cemeteries to SVG

### PR DESCRIPTION
**Pull request by kocio-pl**
Related to https://github.com/gravitystorm/openstreetmap-carto/issues/2045.

Warsaw, z19
Before
d_qk7zl4
After
xq1vlchd

### CODE REVIEW

File: symbols/grave_yard_jewish.svg [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   version="1.1"
+   width="18"
+   height="18"
+   viewBox="0 0 18 18"
```
Comment (pnorman) [inline symbols/grave_yard_jewish.svg]: needs to be a multiple of 2, for the same reasons as #2716

File: symbols/grave_yard_jewish.svg [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+     id="defs8" />
+  <rect
+     width="18"
+     height="18"
+     x="4"
+     y="5"
+     id="canvas"
+     style="visibility:hidden;fill:none;stroke:none" />
+  <path
+     style="fill:#88b78e;fill-opacity:1;stroke-width:0.73192507"
```
Comment (pnorman) [inline symbols/grave_yard_jewish.svg]: It's best to not have stroke

**Comment by kocio-pl** [comment]
I guess stroke was not used anyway, but I removed it. Current PNG version is 18 px, but I've changed it to 16 px and now the pattern is a bit more dense:
Before
o qhp3m6
After
mnylwqrf

**Comment by pnorman** [comment]
The icons look bolder now - is this accurate, or just perceptual?

**Comment by kocio-pl** [comment]
The color is the same and the size is rather similar, but current images are more blurred than new ones.

**Comment by kocio-pl** [comment]
Not needed due to https://github.com/gravitystorm/openstreetmap-carto/pull/2714 work.

**Pull request closed without merging**